src/libkernel/sscp/amdgpu/general_builtins.cpp

```diff
@@ -0,0 +1,21 @@
+/*
+ * This file is part of AdaptiveCpp, an implementation of SYCL and C++ standard
+ * parallelism for CPUs and GPUs.
+ *
+ * Copyright The AdaptiveCpp Contributors
+ *
+ * AdaptiveCpp is released under the BSD 2-Clause "Simplified" License.
+ * See file LICENSE in the project root for full license details.
+ */
+// SPDX-License-Identifier: BSD-2-Clause
+
+#include "hipSYCL/sycl/libkernel/sscp/builtins/builtin_config.hpp"
+#include "hipSYCL/sycl/libkernel/sscp/builtins/amdgpu_general.hpp"
```

Comment:
Not sure "general" is the right name here. Assuming we don't really need fractf, isn't read first lane related to broadcasts, shuffles etc? So maybe put there?